src/Wrapper.js

```diff
@@ -419,30 +483,20 @@ const AppScreen = styled.div`
   overflow: auto;
 `
 
-const LoadingAnimation = styled.img`
-  display: block;
-  margin-bottom: 32px;
-`
-
-const LoadingApps = () => (
-  <div
-    style={{
-      display: 'flex',
-      justifyContent: 'center',
-      alignItems: 'center',
-      height: '100%',
-      flexDirection: 'column',
-    }}
-  >
-    <LoadingAnimation src={ethereumLoadingAnimation} />
-    Loading apps…
-  </div>
-)
-
 export default props => {
   return (
-    <Viewport>
-      {({ below }) => <Wrapper {...props} autoClosingPanel={below('medium')} />}
-    </Viewport>
+    <IdentityConsumer>
```

Comment:
Wonder if hooks would be easier here, no?

Reply:
Yes good idea, let’s move to hooks here! I’ll add an identity hook so that it can be used directly.